docs/chat_managers/botpress_v12.md

```diff
@@ -0,0 +1,66 @@
+# Botpress v12
+
+You can use the AAQ endpoints through various chat managers. Below is an example of how to get Botpress v12 (OSS) running and connected to AAQ endpoints using a provided demo bot.
+
+!!! note "[Botpress v12](https://github.com/botpress/v12/) is open-source and available to self-host but [Botpress Cloud](https://botpress.com/) is a completely different closed-source product."
+
+## Installation
+
+### Via Docker
+
+To install through Docker (recommended), follow the official Botpress v12 docs [here](https://hub.docker.com/r/botpress/server). In short:
+
+1. Get the image
+
+        docker pull botpress/server
+
+2. Run the image
+
+        docker run -d --name=botpress -p 3000:3000 botpress/server
+
+    ??? note "Issues running the Docker image on Macs with Apple Silicon?"
+
+        If you're having issues with image type on MacOS, you can also try running the image through the Docker UI instead of commandline. This fixed the issue for me.
```

Comment:
How about we remove this? It is not actually an issue just a warning!

```suggestion
```

Reply:
Oh yeah you're right - it does run for me too but with a warning